src/main/resources/static/js/index.js

```diff
@@ -1,10 +1,34 @@
-/*
-* Load jquery DataTables on results table (for sorting)
- */
 $(document).ready(function () {
+    // Load jquery DataTables on results table (for sorting)
     $('#results-table').DataTable();
+
+    // Starting jQuery library select2 for searchable select boxes https://select2.org/
+    $('.select2').select2({
+        placeholder: 'Choose...',
+        allowClear: true
+    });
 });
 
+$('#programSelect').change( function() {
```

Comment:
Can you also add some documentation just saying what you're binding this event-handler to. I don't think it's entirely intuitive just from looking at what's going on in here.